test/helpers/utils.go

```diff
@@ -518,6 +520,45 @@ func MCDForNode(cs *framework.ClientSet, node *corev1.Node) (*corev1.Pod, error)
 	return mcdForNode(cs, node)
 }
 
+// Returns the node's uptime expressed as the total number of seconds the
+// system has been up.
+// See: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/deployment_guide/s2-proc-uptime
+func GetNodeUptime(t *testing.T, cs *framework.ClientSet, node corev1.Node) float64 {
+	output := ExecCmdOnNode(t, cs, node, "cat", "/rootfs/proc/uptime")
```

Comment:
Not a blocker but something to address in followup: parsing `/proc/sys/kernel/random/boot_id` is a better version of this, it's already used by the MCD; see `getBootID()`.  That approach will handle the case where e.g. the node is up longer than the first uptime by the time we manage to log back in.